src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 # Database connection properties
 spring.datasource.url=jdbc:postgresql://localhost:5432/test
```

Comment:
Why do you have your database named `test`? It should hold a custom name, related to your application business.